### ISSUE TITLE
AVS audio not correct encoded?

### ISSUE THREAD

**Issue by quamt** [rigaya/QSVEnc#157]
Hello,

I noticed that when I am using AVS script where it reads input file like this
```
v = LWLibavVideoSource("F:\video.mkv")
a = LWLibavAudioSource("F:\video.mkv")
audiodub(v,a)
```

The qsvenc command gets ignored:

```
--audio-codec aac:aac_coder=twoloop
--audio-bitrate 128
--audio-stream :stereo
```

And the audio stereo will not be honored:

```
Output         HEVC(yuv420 10bit) main10 @ Level 5 (high tier)
               1920x1080p 1:1 24.000fps (24/1fps)
               avwriter: hevc, #1:pcm_f32le/6ch -> aac/5.1(side)/128kbps,
                subtitle#1 => matroska
```

Is this a problem with my setting?

**Comment by rigaya** [comment]
Thanks for reporting, seems like audio-stream is ignored for avs input. I'll check and make a fix later.

**Comment by rigaya** [comment]
This should have been fixed with [QSVEnc 7.47](https://github.com/rigaya/QSVEnc/releases/tag/7.47).

**Comment by quamt** [comment]
Thank you.
Seems to work now.
